addons/mail/static/src/new/core_ui/primary_variables.scss

```diff
@@ -13,6 +13,7 @@ $o-mail-thread-typing-icon-size-medium: 5px !default;
 $o-mail-thread-typing-icon-size-small: 3px !default;
 
 $o-mail-message-sidebar-width: 48px !default;
+$navigable-list-z-index: 11;
```

Comment:
I moved in `primary_variables` as to avoid implicit ordering `composer < core_ui` from folder names.